build/webpack/webpack.extension.config.js

```diff
@@ -26,6 +26,12 @@ const config = {
     },
     module: {
         rules: [
+            {
+                loader: 'vscode-nls-dev/lib/webpack-loader',
+                options: {
+                    base: path.join(__dirname, '../../src/client'),
```

Comment:
```suggestion
                    // points to where the entrypoint is (the extension.ts)
                    base: path.join(__dirname, '../../src/client'),
```
nit: a nice comment to explain this.